who_depends_this_lib

```diff
@@ -63,7 +63,29 @@ def extract_package(pkg):
   finally:
     shutil.rmtree(d, onerror=partial(handle_rmtree_error, d))
 
-def check_package(pkg, lib_re):
+def check_package(pkg, lib_re, dep_pkgname):
+  # If a package links to a library that matches `lib_re` but does not have
+  # `dep_pkgname` installed during the build, that package is already broken
+  # For example, packages with files linked to libprotobuf.so should have
+  # protobuf installed during the build.
+  has_dep = False
+  with tarfile.open(pkg) as tar:
+    counter = 0
+    threshold = 10
+    for tarinfo in tar:
+      if counter > threshold:
+        logger.warning('Cannot find .BUILDINFO in first %d entries of %s, broken package?' % (
+          threshold, pkg))
```

Comment:
Don't format the string yourself with `%`. `logging` will format it when outputing (and the orignal format message will be available for the logging system, e.g. Sentry will group them together even when the arguments are different).